### DISCUSSION TITLE
How to make this previewReport faster.

### DISCUSSION THREAD

**Discussion by liuzitong** [fralx/LimeReport#437] [Q&A] [answered]
It's taking like 9 secs for previewReport to show up.I'm think it's because rendered in high resolution.50% scale looks right size.But i dont know how to make resolution lower.Anyone can help?By the way my CPU is i5 3470.

**Comment by fralx** [comment]
Hi, preview rendering time not depends on resolution. It depends on the report complexity

**Reply by liuzitong** [reply, marked as answer]
I reccently just find out reason.It's because it was enumerating printer devices before open preview windows.After i deleted some devices in windows system,it only takes about 1 second.Thanks for answering by the way.Your lib is very good.
